examples/Polls Hypermedia API.md

```diff
@@ -0,0 +1,408 @@
+            },
+            "_embedded": {
+                "question": [
+                    {
+                        "_links": {
+                            "self": { "self": "\/questions\/1" }
+                        },
+                        "_embedded": {
+                            "choice": [
+                                {
+                                    "_links": {
+                                        "self": { "self": "\/questions\/1\/choices\/1" }
+                                    },
+                                    "choice": "Swift",
+                                    "votes": 2048
+                                }, {
+                                    "_links": {
+                                        "self": { "self": "\/questions\/1\/choices\/2" }
+                                    },
+                                    "choice": "Python",
+                                    "votes": 1024
+                                }, {
+                                    "_links": {
+                                        "self": { "self": "\/questions\/1\/choices\/3" }
+                                    },
+                                    "choice": "Objective-C",
+                                    "votes": 512
+                                }, {
+                                    "_links": {
+                                        "self": { "self": "\/questions\/1\/choices\/4" }
+                                    },
+                                    "choice": "Ruby",
+                                    "votes": 256
+                                }
+                            ]
+                        },
+                        "question": "Favourite programming language?",
+                        "published_at": "2014-11-11T08:40:51.620Z"
+                    }
+                ]
+            }
+        }
+
+### Create a new question [POST]
+
+You can create your own question using this action. It takes a JSON dictionary containing a question and a collection of answers in the form of choices.
+
+- question (string) - The question
+- choices (array[string]) - A collection of choices.
+
++ Request (application/json)
+
+            {
+                "question": "Favourite programming language?",
+                "choices": [
+                    "Swift",
+                    "Python",
+                    "Objective-C",
+                    "Ruby"
+                ]
+            }
+
++ Response 201
+
+    [Question][]
+
+## Group Question
+
+Resource related to questions in the API.
+
+## Question [/questions/{question_id}]
+
+A Question object has the following attributes:
+
+- question
+- published_at - An ISO8601 date when the question was published.
+- url
+- choices - An array of Choice objects.
+
++ Parameters
+    + question_id (required, number, `1`) ... ID of the Question in form of an integer
+
++ Model (application/vnd.siren+json)
```

Comment:
As I want to deprecate and remove the Model concept entirely (to be replaced with Attributes) I would not use it in this tutorial..